webknossos/format.sh

```diff
@@ -2,9 +2,8 @@
 set -eEuo pipefail
 
 if [ $# -eq 1 ] && [ "$1" = "check" ]; then
-    poetry run black --check .
-    poetry run isort --check-only .
+    poetry run ruff format --check .
 else
-    poetry run isort .
-    poetry run black .
+    poetry run ruff check --fix .
```

Comment:
This is my replacement for `isort`, i.e. automatic fixing of the import linting errors. This, however, will list and fix (if possible) any linting errors. It is fast so why not...

Reply:
Does `poetry run ruff check --select I --fix .` work? copied from [here](https://docs.astral.sh/ruff/formatter/#sorting-imports). I think, I would prefer to keep formatting and linting separated if easily possible.

Reply:
For an earlier version I had the same idea and this works. I applied your suggestion on my latest commit.